FormalConjectures/ErdosProblems/12.lean

```diff
@@ -0,0 +1,77 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+
+import FormalConjectures.Util.ProblemImports
+
+/-!
+# Erdős Problem 12
+
+*Reference:* [erdosproblems.com/12](https://www.erdosproblems.com/12)
+-/
+
+open Classical
+
+-- TODO: add other statements from the file
```

Comment:
Could you also create a corresponding issue stating this (unless you have already!)

Reply:
Added a comment to the Erdos Problem 12 issue